frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepage.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import type { Meta, StoryObj } from "@storybook/react";
```

Comment:
This may or may not land in master ([slack](https://metaboat.slack.com/archives/C505ZNNH4/p1710781977025789)), but for now it's helpful to test all the variants of the UI.

Reply:
I have no objection for having this in the repo if it helps with feature development.